### PR TITLE
Create ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-53597](https://cumulocity.atlassian.net/browse/MTM-53597)
Original PR: [5601](https://github.softwareag.com/IOTA/cumulocity-ui/pull/5601)

### CODE REVIEW

File: content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+date: ""
+title: Allowing more links on login page and show it on all public facing pages
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md:3]: Please use sentence capitalization: Additional links on Login page visible across public pages
Also, I´m not sure if this is what the change is about. Please check.

File: content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+ - allows to add more than one link on the login page
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md:5]: Is this part of the title? Should be removed

File: content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-53597
+version: 1019.6.5
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md:18]: @janhommes Is that correct? Must be "Login page" (2x)

**Review by janhommes — APPROVED** [review]

File: content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+---
+This update enables adding multiple links to the login page by a custom application configuration. These links will be displayed on all public-facing pages of the site, not only on the login page. This provides a consistent experience for users accessing public pages, with easy access to extra helpful links from any page. You can for example add there a privacy notice or a terms and conditions link.
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md:17]:
```suggestion
This update enables adding multiple links to the Login page by a custom application configuration. These links will be displayed on all public-facing pages of the site, not only on the Login page. This provides a consistent experience for users accessing public pages, with easy access to extra helpful links from any page. You can add a privacy notice, for example, or a terms and conditions link.

```

File: content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Allowing more links on the login page and showing it on all public-facing pages
```
Comment (BeateRixen) [inline content/change-logs/ui-c8y-1019-6-5-allowing-more-links-on-login-page-and-show-it-on-all-public-facing-pages-allows-to-add-more-than-one-link-on-the-login-page-those-links-are-shown-all-public-facing-pages-not-only-on-the-login-page.md:3]:
```suggestion
title: Enabling multiple links on the Login page which are shown on all public-facing pages

```

**Comment by BeateRixen** [comment]
In this case I would recommend to change the file title as it is quite different from the actual title now

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**